### PR TITLE
Review-of-one-of-my-projects

### PR DESCRIPTION
**Pull request by EbukaNweje**
Adding Files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+  <body>
+        <h1>Hello World, we are software developers!</h1>
+        <nav id="navbar">
+          <header>JS Documentation</header>
+          <ul>
+            <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+            <li>
+              <a class="nav-link" href="#What_you_should_already_know"
+                >What you should already know</a
+              >
```
Comment (ezimahebuka) [inline index.html:18]: Please consider closing this `li` tag for your code to be in compliance with HTML best practices. e.g 
`<li><a href="#">What you should already know</a></li>`

Comment (muneebulrehman) [reply index.html:18]: Hey, let us not be harsh, Remember, it is always important to be polite and nice.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @EbukaNweje 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

- Always appreciate the person for what he did well before requesting for the changes to be made.
- Your review should have a body or a description where you detail what is good in the code. Like this one, you are reading now.
- You can leave a comment or two about the readme file because it is essentially empty. There is still some scope on which you can review the project further.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+              <p>
+                JavaScript is a very free-form language compared to Java. You do not
+                have to declare all variables, classes, and methods. You do not have to
+                be concerned with whether methods are public, private, or protected, and
+                you do not have to implement interfaces. Variables, parameters, and
+                function return types are not explicitly typed.
+              </p>
+            </article>
+          </section>
+          <section class="main-section" id="Hello_world">
```
[the remark this answers is not in the document]
Comment (EbukaNweje) [reply index.html:129]: Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

[OPTIONAL] <section class="main-section">

**Review by EbukaNweje — COMMENTED** [review]
Adding Comment

File: index.html (hunk 1 above)
Comment (danmainah) [reply index.html:18]: - [ ] Good job @EbukaNweje nice adding polite words in your review, don't forget to point out the issue than needs to be fixed clearly, it took me some time to understand the mistake that you are pointing out.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @EbukaNweje , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️  Use of polite language.
✔️  Good use of inline comments.

 # Required Changes ♻️

- [ ] Kindly make sure to add a review summary in your review, you can create another review in this pull request implement the changes requested by adding one or two inline comments, and then add a project summary, below is an image on the top right corner you will see a finish your review tap click it to reveal a tap to write your summary
image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EbukaNweje — COMMENTED** [review]
## Required changes ♻️ 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there ! 
Please see my inline comments for the requested changes.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+            <header>Hello world</header>
+            <article>
+              To get started with writing JavaScript, open the Scratchpad and write your
+              first "Hello world" JavaScript code:
+              <code
+                >function greetMe(yourName) { alert("Hello " + yourName); }
+                greetMe("World");
```
Comment (ezimahebuka) [inline index.html:136]: Please consider closing the `code` tag for your code to be in compliance with HTML best practices. e.g
`<code>function greetMe(yourName) { alert("Hello " + yourName); } greetMe("World"); </code>`

**Review by EbukaNweje — COMMENTED** [review]
Hi, @EbukaNweje,
 Excellent job so far 👍 but a few changes still need to be addressed. Kindly check the inline comment

**Review by EbukaNweje — COMMENTED** [review]
Required changes ♻️
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Please see my inline comments for the requested changes.

**Review by danmainah — APPROVED** [review]
Hi @EbukaNweje , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Good use of inline comment
✔️  Good use of simple and polite messages.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**